pkg/controller/util/node/controller_utils.go

```diff
@@ -463,3 +463,49 @@ func ValidateUrl(urlString string) error {
 
 	return nil
 }
+
+// TODO: refactor those shared functions: Arktos issue #997
+//
+func GetTenantPartitionClientsFromFiles(kubeconfigFiles []string) ([]clientset.Interface, error) {
```

Comment:
This code is merging to master, please use existing util now, don't let refactor work lingering in master.

Reply:
originally the plan is to address those code level reuse in issue 997. since it seems there are some concerns to get this into the mater branch with the issue open. i have make initial fix to it and committed to this PR.